scripts/vote_2023_10_03.py

```diff
@@ -0,0 +1,70 @@
+"""
+Voting 03/10/2023.
+
+1. Update Anchor Vault implementation from 0x07BE9BB2B1789b8F5B2f9345F18378A8B036A171 to #TBA
+2. Finalize Anchor Vault upgrade
+
+Vote passed & executed on #, block #
+"""
+
+import time
+
+from typing import Dict, Tuple, Optional
+
+from brownie.network.transaction import TransactionReceipt
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+from utils.agent import agent_forward
+
+from utils.config import (
+    get_deployer_account,
+    contracts,
+    get_is_live,
+    get_priority_fee,
+)
+
+ANCHOR_NEW_ADDRESS = "#TBA"
+
+def start_vote(tx_params: Dict[str, str], silent: bool = False) -> Tuple[int, Optional[TransactionReceipt]]:
+
+    call_script_items = [
+        # 1. Update Anchor Vault implementation
+        agent_forward(
+            [
+                (
+                    contracts.anchor_vault_proxy.address,
+                    contracts.anchor_vault_proxy.proxy_upgradeTo.encode_input(ANCHOR_NEW_ADDRESS, b""),
```

Comment:
I am just wondering: we might have passed non-empty data to call `finalize_upgrade_v4` as a part of the `proxy_upgradeTo` invocation (see https://github.com/lidofinance/anchor-collateral-steth/blob/main/contracts/AnchorVaultProxy.sol#L74) moving with a single vote item.

Is it better or worse?

Reply:
dont get it

Reply:
when you call `proxy_upgradeTo` instead of empty data you can pass the second argument encoded to call an initializer (`finalize_upgrade_v4`)

Reply:
yep, lets go with single vote item